apps/blade/src/app/admin/events/_components/delete-event.tsx

```diff
@@ -54,7 +55,11 @@ export function DeleteEventButton({ event }: DeleteEventButtonProps) {
 
   const handleDelete = () => {
     setIsLoading(true);
-    deleteEvent.mutate({ id: event.id, discordId: event.discordId });
+    deleteEvent.mutate({
```

Comment:
unrelated to this commit, but why is extraCaution not const?

Reply:
esLint would blow me up if i made it a const, since technically it isnt changed. wonder how else to define a flag... i may move to consts

Reply:
wdym, since it doesn't change, why is eslint blowing up